yellow-paper/docs/state/archive.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Archive
+---
+
+# Archive
+
+The Archive is an [append-only Merkle tree](./tree_impls.md#append-only-merkle-trees) that stores the headers of all previous blocks in the chain as its leaves. 
+
+For most chains this is not required since they are always executing at the head of the chain. However, since private execution relies on proofs generated by the user, they don't have the current head and must be based on historic state. By including all prior headers (which include commitments to the state) the Archive allows us to easily prove that the historic state that a transaction is proven upon was valid.
```

Comment:
```suggestion
For most chains this is not required since they are always executing at the head of the chain. However, private execution relies on proofs generated by the user, and since users don't know the current head (only the sequencer does) they must base their proofs on historic state. By including all prior headers (which include commitments to the state) the Archive allows us to easily prove that the historic state that a transaction was proven upon is valid.
```